cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html

```diff
@@ -64,72 +65,22 @@ <h2 class="header-slug">
                     </figure>
                 {% endif %}
             {% endfor %}
+        {% endif %}
 
-            <h3>{{ page.contact.heading }}</h3>
-
-            <p>{{ page.contact.body |safe }}</p>
-        </div>
+        {% if value.header %}
+            <h3>{{ value.header }}</h3>
+        {% endif %}
 
-        <div class="o-sidebar-contact-info_column">
-            {% if page.contact.contact_info %}
-                {% for block in page.contact.contact_info %}
-                    {{ render_stream_child(block) }}
-                {% endfor %}
-            {% endif %}
-        </div>
+        {{ value.contact.body | safe }}
```

Comment:
@KimberlyMunoz nah this `body` field is within the contact model which is required for the organism. but that brings up another point this organism also has its own body field `value.body` thats never used in this template, the same goes for `main-contact-info` is this something we want to render? or remove?